framework/console/controllers/MessageController.php

```diff
@@ -527,7 +527,9 @@ protected function extractMessagesFromTokens(array $tokens, array $translatorTok
                                 $fullMessage = mb_substr($buffer[2][1], 1, -1);
                                 $i = 3;
                                 while ($i < count($buffer) - 1 && !is_array($buffer[$i]) && $buffer[$i] === '.') {
-                                    $fullMessage .= mb_substr($buffer[$i + 1][1], 1, -1);
+                                    if (isset($buffer[$i+1], $buffer[$i+1][1])) {
+                                        $fullMessage .= mb_substr($buffer[$i + 1][1], 1, -1);
+                                    }
                                     $i += 2;
```

Comment:
I got a `Uninitialized string offset: 1` in a special constellation i have in my project. This line ensures that he skips the failing translation instead of breaking the whole process.

Unfortunately we can't do this already in the isBufferValid() method since we need the $i loop counter.

Reply:
@thyseus do you know which "special constellation" that is? could you share the code?

Reply:
https://gist.github.com/thyseus/837e7f541dd89c4e64a0d4391e79305e